.pipelines/templates/template-rp-and-billing-e2e-section.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+    last_stage_of_previous_section: ""
+    e2e_subscription: ""
+    aro_v4_e2e_devops_spn: ""
+    billingE2EPipelineName: ""
+    billingE2EBranchName: ""
+    gobin: ""
+    gopath: ""
+    goroot: ""
+    modulePath: ""
+    locations: []
```

Comment:
is it possible to homogenise the parameter names with those in template-rp-and-billing-e2e.yml ?

Reply:
Done